server/models/List.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require("mongoose");
+
+const listSchema = new mongoose.Schema({
+    name: {
+      type: String,
+      required: true,
+      unique: true
```

Comment:
This will cause a problem if two users want to give their list the same name

Reply:
fixed